Calculator/Calculator/Controller/ViewController.swift

```diff
@@ -82,34 +81,50 @@ final class ViewController: UIViewController {
     //MARK: - IBAciton
     @IBAction private func operandsButtonTapped(_ sender: UIButton) {
         guard let number = sender.currentTitle,
-              currentOperand.count < 19 else { return }
-        
+              currentOperand.filter({ Int(String($0)) != nil }).count < 20 else { return }
+
```

Comment:
고차함수 `filter` 를 잘 활용해 주셨네요 👏
```suggestion
              currentOperand.filter { Int(String($0)) != nil }.count < 20 else { return }
```

이렇게 소괄호를 생략해줘도 좋을 것 같아요 ☺️

Reply:
수정 완료했습니다!